tests/apple-exporter/test-class-exporter-content.php

```diff
@@ -32,5 +32,13 @@ public function testCompleteContent() {
 		$this->assertEquals( 'someurl.com', $content->cover() );
 	}
 
-}
+	public function test_format_src_url() {
+		$content = new \Apple_Exporter\Exporter_Content( 13, 'Title', '<p>Example content</p>' );
```

Comment:
Since the function is static, you don't need an instance of the class to run the test. It should just be:

```php
$this->assertEquals(
    'https://example.com/some.mp3?one=two&query=arg',
    \Apple_Exporter\Exporter_Content::format_src_url( 'https://example.com/some.mp3?one=two&amp;query=arg' )
);
```